plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/UiSessionTimeout.js

```diff
@@ -155,7 +155,10 @@ Ext.define('NX.controller.UiSessionTimeout', {
       run: function (count) {
         win.down('label').setText('Session will expire in ' + (me.SECONDS_TO_EXPIRE - count) + ' seconds');
         if (count === me.SECONDS_TO_EXPIRE) {
-          win.close();
+          win.down('label').setText('Your session has expired. Please log in.');
```

Comment:
Wanted to get the other I18n-related PRs merged before tackling this. Will do before merge.

/me hates merge conflicts